node_normalizer/server.py

```diff
@@ -72,42 +88,80 @@ async def shutdown_event():
 
 
 @app.post(
-    f'/response',
-    summary='Normalizes a TRAPI response object',
-    description='Returns the response object with a merged '
-                'knowledge graph and query graph bindings'
+    f"/query",
+    summary="Normalizes a TRAPI response object",
+    description="Returns the response object with a merged "
+    "knowledge graph and query graph bindings",
 )
-async def normalize_response(response: Response) -> Response:
+async def query(response: Response) -> Response:
     """
     Normalizes a TRAPI compliant knowledge graph
     """
     response.message = await normalize_message(app, response.message)
     return response
 
 
+@app.post(
+    f"/asyncquery",
+    summary="Normalizes a TRAPI response object",
+    description="Returns the response object with a merged "
+    "knowledge graph and query graph bindings",
+)
+async def async_query(response: Response, callback: str):
```

Comment:
I don't think this is quite right.  the callback is not a parameter, instead it is a field in response. So it'll be like:
```
{
   "callback": "http://example.com/whatever",
   "message": {...}
}

Reply:
it is possible to test the callback.

if working from hoe you can configure your home router to port forward to go to a debug session running in your local environment.

Reply:
Yes, this is correct.  But is there a way to do it nicely in an automated way, e.g. as part of the travis-ci etc...